pom.xml

```diff
@@ -58,13 +58,10 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.3.0</cdap.version>
-    <hydrator.version>1.8.0</hydrator.version>
-    <widgets.dir>widgets</widgets.dir>
-    <docs.dir>docs</docs.dir>
-    <!-- properties for script build step that creates the config files for the artifacts -->
-    <app.parents>system:cdap-data-pipeline[4.3.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-streams[4.3.0-SNAPSHOT,5.0.0-SNAPSHOT)
-    </app.parents>
+    <cdap.version>5.0.0</cdap.version>
```

Comment:
This should match the lowest compatible pipeline version. Otherwise somebody could, for example, add field level lineage code, making it incompatible with 4.3.0.